SocialSharing/SocialSharingViewController.m

```diff
@@ -12,6 +12,7 @@ @interface SocialSharingViewController () {
     UIImagePickerController *pickerfromPhotoLibrary;
     UIImagePickerController *pickerfromCamera;
```

Comment:
Why two UIImagePickerController? May be we can handle by one just changing the sourcetype.
Do we need set it globally ? Was is it necessary?